### PR TITLE
Change CSS for Change Visibility , closes #1045

### LINKED ISSUE #1045: Change CSS for Change Visibility
The current design:

Bildschirmfoto 2024-09-13 um 08 33 56

Let's do something like this:

Bildschirmfoto 2024-09-13 um 08 42 44

Display a button to the options which are available and don't display the button for the active one.

Here's the HTML/CSS:

    <ul role="list" class="divide-y divide-gray-100">
      <li class="flex items-center justify-between gap-x-6 py-5">
        <div class="min-w-0">
          <div class="flex items-start gap-x-3">
            <p class="text-sm font-semibold leading-6 text-gray-900">GraphQL API</p>
            <p class="mt-0.5 whitespace-nowrap rounded-md bg-green-50 px-1.5 py-0.5 text-xs font-medium text-green-700 ring-1 ring-inset ring-green-600/20">Complete</p>
          </div>
          <div class="mt-1 flex items-center gap-x-2 text-xs leading-5 text-gray-500">
            <p class="whitespace-nowrap">Due on <time datetime="2023-03-17T00:00Z">March 17, 2023</time></p>
            <svg viewBox="0 0 2 2" class="h-0.5 w-0.5 fill-current">
              <circle cx="1" cy="1" r="1" />
            </svg>
            <p class="truncate">Created by Leslie Alexander</p>
          </div>
        </div>
        <div class="flex flex-none items-center gap-x-4">
          <a href="#" class="hidden rounded-md bg-white px-2.5 py-1.5 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50 sm:block">View project<span class="sr-only">, GraphQL API</span></a>
          <div class="relative flex-none">
            <button type="button" class="-m-2.5 block p-2.5 text-gray-500 hover:text-gray-900" id="options-menu-0-button" aria-expanded="false" aria-haspopup="true">
              <span class="sr-only">Open options</span>
              <svg class="h-5 w-5" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                <path d="M10 3a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM10 8.5a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM11.5 15.5a1.5 1.5 0 10-3 0 1.5 1.5 0 003 0z" />
              </svg>
            </button>

            
            <div class="absolute right-0 z-10 mt-2 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-gray-900/5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="options-menu-0-button" tabindex="-1">
              
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-0-item-0">Edit<span class="sr-only">, GraphQL API</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-0-item-1">Move<span class="sr-only">, GraphQL API</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-0-item-2">Delete<span class="sr-only">, GraphQL API</span></a>
            </div>
          </div>
        </div>
      </li>
      <li class="flex items-center justify-between gap-x-6 py-5">
        <div class="min-w-0">
          <div class="flex items-start gap-x-3">
            <p class="text-sm font-semibold leading-6 text-gray-900">New benefits plan</p>
            <p class="mt-0.5 whitespace-nowrap rounded-md bg-gray-50 px-1.5 py-0.5 text-xs font-medium text-gray-600 ring-1 ring-inset ring-gray-500/10">In progress</p>
          </div>
          <div class="mt-1 flex items-center gap-x-2 text-xs leading-5 text-gray-500">
            <p class="whitespace-nowrap">Due on <time datetime="2023-05-05T00:00Z">May 5, 2023</time></p>
            <svg viewBox="0 0 2 2" class="h-0.5 w-0.5 fill-current">
              <circle cx="1" cy="1" r="1" />
            </svg>
            <p class="truncate">Created by Leslie Alexander</p>
          </div>
        </div>
        <div class="flex flex-none items-center gap-x-4">
          <a href="#" class="hidden rounded-md bg-white px-2.5 py-1.5 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50 sm:block">View project<span class="sr-only">, New benefits plan</span></a>
          <div class="relative flex-none">
            <button type="button" class="-m-2.5 block p-2.5 text-gray-500 hover:text-gray-900" id="options-menu-1-button" aria-expanded="false" aria-haspopup="true">
              <span class="sr-only">Open options</span>
              <svg class="h-5 w-5" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                <path d="M10 3a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM10 8.5a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM11.5 15.5a1.5 1.5 0 10-3 0 1.5 1.5 0 003 0z" />
              </svg>
            </button>

            
            <div class="absolute right-0 z-10 mt-2 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-gray-900/5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="options-menu-1-button" tabindex="-1">
              
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-1-item-0">Edit<span class="sr-only">, New benefits plan</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-1-item-1">Move<span class="sr-only">, New benefits plan</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-1-item-2">Delete<span class="sr-only">, New benefits plan</span></a>
            </div>
          </div>
        </div>
      </li>
      <li class="flex items-center justify-between gap-x-6 py-5">
        <div class="min-w-0">
          <div class="flex items-start gap-x-3">
            <p class="text-sm font-semibold leading-6 text-gray-900">Onboarding emails</p>
            <p class="mt-0.5 whitespace-nowrap rounded-md bg-gray-50 px-1.5 py-0.5 text-xs font-medium text-gray-600 ring-1 ring-inset ring-gray-500/10">In progress</p>
          </div>
          <div class="mt-1 flex items-center gap-x-2 text-xs leading-5 text-gray-500">
            <p class="whitespace-nowrap">Due on <time datetime="2023-05-25T00:00Z">May 25, 2023</time></p>
            <svg viewBox="0 0 2 2" class="h-0.5 w-0.5 fill-current">
              <circle cx="1" cy="1" r="1" />
            </svg>
            <p class="truncate">Created by Courtney Henry</p>
          </div>
        </div>
        <div class="flex flex-none items-center gap-x-4">
          <a href="#" class="hidden rounded-md bg-white px-2.5 py-1.5 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50 sm:block">View project<span class="sr-only">, Onboarding emails</span></a>
          <div class="relative flex-none">
            <button type="button" class="-m-2.5 block p-2.5 text-gray-500 hover:text-gray-900" id="options-menu-2-button" aria-expanded="false" aria-haspopup="true">
              <span class="sr-only">Open options</span>
              <svg class="h-5 w-5" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                <path d="M10 3a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM10 8.5a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM11.5 15.5a1.5 1.5 0 10-3 0 1.5 1.5 0 003 0z" />
              </svg>
            </button>

            
            <div class="absolute right-0 z-10 mt-2 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-gray-900/5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="options-menu-2-button" tabindex="-1">
              
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-2-item-0">Edit<span class="sr-only">, Onboarding emails</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-2-item-1">Move<span class="sr-only">, Onboarding emails</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-2-item-2">Delete<span class="sr-only">, Onboarding emails</span></a>
            </div>
          </div>
        </div>
      </li>
      <li class="flex items-center justify-between gap-x-6 py-5">
        <div class="min-w-0">
          <div class="flex items-start gap-x-3">
            <p class="text-sm font-semibold leading-6 text-gray-900">iOS app</p>
            <p class="mt-0.5 whitespace-nowrap rounded-md bg-gray-50 px-1.5 py-0.5 text-xs font-medium text-gray-600 ring-1 ring-inset ring-gray-500/10">In progress</p>
          </div>
          <div class="mt-1 flex items-center gap-x-2 text-xs leading-5 text-gray-500">
            <p class="whitespace-nowrap">Due on <time datetime="2023-06-07T00:00Z">June 7, 2023</time></p>
            <svg viewBox="0 0 2 2" class="h-0.5 w-0.5 fill-current">
              <circle cx="1" cy="1" r="1" />
            </svg>
            <p class="truncate">Created by Leonard Krasner</p>
          </div>
        </div>
        <div class="flex flex-none items-center gap-x-4">
          <a href="#" class="hidden rounded-md bg-white px-2.5 py-1.5 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50 sm:block">View project<span class="sr-only">, iOS app</span></a>
          <div class="relative flex-none">
            <button type="button" class="-m-2.5 block p-2.5 text-gray-500 hover:text-gray-900" id="options-menu-3-button" aria-expanded="false" aria-haspopup="true">
              <span class="sr-only">Open options</span>
              <svg class="h-5 w-5" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                <path d="M10 3a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM10 8.5a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM11.5 15.5a1.5 1.5 0 10-3 0 1.5 1.5 0 003 0z" />
              </svg>
            </button>

            
            <div class="absolute right-0 z-10 mt-2 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-gray-900/5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="options-menu-3-button" tabindex="-1">
              
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-3-item-0">Edit<span class="sr-only">, iOS app</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-3-item-1">Move<span class="sr-only">, iOS app</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-3-item-2">Delete<span class="sr-only">, iOS app</span></a>
            </div>
          </div>
        </div>
      </li>
      <li class="flex items-center justify-between gap-x-6 py-5">
        <div class="min-w-0">
          <div class="flex items-start gap-x-3">
            <p class="text-sm font-semibold leading-6 text-gray-900">Marketing site redesign</p>
            <p class="mt-0.5 whitespace-nowrap rounded-md bg-yellow-50 px-1.5 py-0.5 text-xs font-medium text-yellow-800 ring-1 ring-inset ring-yellow-600/20">Archived</p>
          </div>
          <div class="mt-1 flex items-center gap-x-2 text-xs leading-5 text-gray-500">
            <p class="whitespace-nowrap">Due on <time datetime="2023-06-10T00:00Z">June 10, 2023</time></p>
            <svg viewBox="0 0 2 2" class="h-0.5 w-0.5 fill-current">
              <circle cx="1" cy="1" r="1" />
            </svg>
            <p class="truncate">Created by Courtney Henry</p>
          </div>
        </div>
        <div class="flex flex-none items-center gap-x-4">
          <a href="#" class="hidden rounded-md bg-white px-2.5 py-1.5 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50 sm:block">View project<span class="sr-only">, Marketing site redesign</span></a>
          <div class="relative flex-none">
            <button type="button" class="-m-2.5 block p-2.5 text-gray-500 hover:text-gray-900" id="options-menu-4-button" aria-expanded="false" aria-haspopup="true">
              <span class="sr-only">Open options</span>
              <svg class="h-5 w-5" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                <path d="M10 3a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM10 8.5a1.5 1.5 0 110 3 1.5 1.5 0 010-3zM11.5 15.5a1.5 1.5 0 10-3 0 1.5 1.5 0 003 0z" />
              </svg>
            </button>

            
            <div class="absolute right-0 z-10 mt-2 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-gray-900/5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="options-menu-4-button" tabindex="-1">
              
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-4-item-0">Edit<span class="sr-only">, Marketing site redesign</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-4-item-1">Move<span class="sr-only">, Marketing site redesign</span></a>
              <a href="#" class="block px-3 py-1 text-sm leading-6 text-gray-900" role="menuitem" tabindex="-1" id="options-menu-4-item-2">Delete<span class="sr-only">, Marketing site redesign</span></a>
            </div>
          </div>
        </div>
      </li>
    </ul>

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
Screenshot 2024-10-16 at 12 18 53
Screenshot 2024-10-16 at 12 19 01

### CODE REVIEW

**Review by wintermeyer — CHANGES_REQUESTED** [review]
I should have made it more clear in the issue. Please have a look at this screenshot:

367176992-b8fd9252-e58f-424e-a8aa-3c8f6f6aee01

The red one is a button. I'd like to have the buttons there. The blue one is a pill. If you want you can add one `active` pill to the currently active state.

**Comment by MICHAELMUNAVU83** [comment]
Okay , thanks for this .
I have 2 questions
Should the "active" state have a button and if so what should it read ?
What should the buttons for the "inactive"  buttons read?

**Comment by wintermeyer** [comment]
> Okay , thanks for this . I have 2 questions Should the "active" state have a button and if so what should it read ?

The active state should not have a button but a green pill (see the blue rectangle) which says `active`.

> What should the buttons for the "inactive" buttons read?

Alle inactive or deactivated buttons read the same as the active buttons. They are just not usable and have a slightly different color.

**Comment by MICHAELMUNAVU83** [comment]
Thanks for this , in the screenshot shared above , the button in red has the name "View Project" what  should ours  have now?
> Alle inactive or deactivated buttons read the same as the active buttons. They are just not usable and have a slightly different color.

**Comment by wintermeyer** [comment]
Ahhh! My bad. Name it „activate“.

**Comment by MICHAELMUNAVU83** [comment]
Thanks , finally for the delete card , should we leave it purely red and should it have the "activate" button or not?

**Comment by wintermeyer** [comment]
Activate for that button doesn’t make sense. Name it Delete account and make it red.

**Comment by MICHAELMUNAVU83** [comment]
We now have it as 

Screenshot 2024-10-16 at 14 28 55
Screenshot 2024-10-16 at 14 29 14

File: lib/animina_web/components/profile/profile_components.ex [hunk 1]
```diff
@@ -598,15 +615,36 @@ defmodule AniminaWeb.ProfileComponents do
...
+    <% end %>
+    """
+  end
+
+  defp inactive_mark(assigns) do
+    ~H"""
+    <div id={"inactive-mark-#{@value}"}>
+      <p class="mt-0.5 whitespace-nowrap rounded-md bg-gray-50 px-1.5 py-0.5 text-xs font-medium text-gray-600 ring-1 ring-inset ring-gray-500/10">
+        <%= with_locale(@language, fn -> %>
+          <%= gettext("Non Active") %>
```
Comment (wintermeyer) [inline lib/animina_web/components/profile/profile_components.ex]: Please remove the `Non Active` pill.

**Review by wintermeyer — CHANGES_REQUESTED** [review]
Is there a way to make the `Activate` and the `Delete Account` buttons the same width? That would be easier on the eyes.

**Comment by wintermeyer** [comment]
All the text should align on the same virtual left line (the red one). Right now there is an indention which doesn't make sense.

Bildschirmfoto 2024-10-16 um 15 40 26

File: lib/animina_web/components/profile/profile_components.ex (hunk 1 above)
Comment (MICHAELMUNAVU83) [reply lib/animina_web/components/profile/profile_components.ex]: I have removed this, and we can also have them same width

**Comment by MICHAELMUNAVU83** [comment]
We now have this page as
Screenshot 2024-10-17 at 09 48 42

**Review by wintermeyer — APPROVED** [review]

**Pull request merged**